sdcm/sct_runner.py

```diff
@@ -840,8 +840,8 @@ def list_sct_runners(cls) -> list[SctRunnerInfo]:
         azure_service = AzureService()
         sct_runners = []
         for instance in list_instances_azure(tags_dict={"NodeType": cls.NODE_TYPE}, verbose=True):
-            if "launch_time" not in instance.tags:
-                LOGGER.warning("Skip Azure instance w/o `launch_time' tag: %s", instance.name)
+            if not instance.tags.get("launch_time"):
```

Comment:
@enaydanov why do we want to skip SCT Runner instances without launch_time?

Reply:
This was the original logic before my refactoring.